packages/@okta/vuepress-site/docs/release-notes/2021/index.md

```diff
@@ -4,6 +4,30 @@ title: Okta API Products Release Notes 2021
 
 ## May
 
+### Weekly Release 2021.05.1
+
+| Change                                                                                              | Expected in Preview Orgs |
+|-----------------------------------------------------------------------------------------------------|--------------------------|
+| [Okta Sign-In Widget, version 5.6.1](#okta-sign-in-widget-version-5-6-1) | May 12, 2021 |
+| [Bugs fixed in 2021.05.1](#bugs-fixed-in-2021-05-1)         | May 12, 2021           |
+
+#### Okta Sign-In Widget, version 5.6.1
+
+For details about this release, see the Okta [Sign-In Widget Release Notes](https://github.com/okta/okta-signin-widget/releases/tag/okta-signin-widget-5.6.1). For more information about the Widget, see the Okta [Sign-In Widget Guide](/code/javascript/okta_sign-in_widget/). <!--OKTA-393866-->
+
+#### Bugs fixed in 2021.05.1
+
+* Duplicate parameter names were passed in requests to the following [OAuth endpoints](/docs/reference/api/oidc/#possible-errors), and no error message was sent. (OKTA-132318)
```

Comment:
Suggested change:
Duplicate parameter names were passed when requests were made to the following OAuth endpoints...

Reply:
@christinechoy-okta - the duplicate parameter names were passed IN the requests, not when requests were made. So, the original sentence is more accurate.